test/src/main/java/org/apache/accumulo/test/fate/zookeeper/FateIT.java

```diff
@@ -71,35 +76,63 @@ public TestOperation(NamespaceId namespaceId, TableId tableId) {
 
     @Override
     public long isReady(long tid, Manager manager) throws Exception {
-      return Utils.reserveNamespace(manager, namespaceId, tid, false, true, TableOperation.RENAME)
-          + Utils.reserveTable(manager, tableId, tid, true, true, TableOperation.RENAME);
+      LOG.info("Entering isReady {}", Long.toHexString(tid));
+      try {
+        FateIT.inReady();
+        return Utils.reserveNamespace(manager, namespaceId, tid, false, true, TableOperation.RENAME)
+            + Utils.reserveTable(manager, tableId, tid, true, true, TableOperation.RENAME);
+      } finally {
+        LOG.info("Leaving isReady {}", Long.toHexString(tid));
+      }
     }
 
     @Override
     public void undo(long tid, Manager manager) throws Exception {
-      Utils.unreserveNamespace(manager, namespaceId, tid, false);
-      Utils.unreserveTable(manager, tableId, tid, true);
+      LOG.info("Entering undo {}", Long.toHexString(tid));
+      try {
+        Utils.unreserveNamespace(manager, namespaceId, tid, false);
+        Utils.unreserveTable(manager, tableId, tid, true);
+      } finally {
+        LOG.info("Leaving undo {}", Long.toHexString(tid));
+      }
     }
 
     @Override
-    public Repo<Manager> call(long tid, Manager environment) throws Exception {
-      FateIT.inCall();
-      return null;
+    public Repo<Manager> call(long tid, Manager manager) throws Exception {
+      LOG.info("Entering call {}", Long.toHexString(tid));
+      try {
+        FateIT.inCall();
+        return null;
+      } finally {
+        Utils.unreserveNamespace(manager, namespaceId, tid, false);
+        Utils.unreserveTable(manager, tableId, tid, true);
+        LOG.info("Leaving call {}", Long.toHexString(tid));
```

Comment:
You have a bunch of `info` statements like this. Could probably log them at debug or remove them if you don't need them anymore.

Reply:
Addressed in c9e64f4